### ISSUE TITLE
lanuch nccl kernels with more than 32 blocks 

### ISSUE THREAD

**Issue by hamid-ramezani** [NVIDIA/nccl#455]
Hello, 

I was wondering if it is possible to launch nccl kernels with more than 32 blocks? If so, how?

**Comment by sjeaugey** [comment]
Probably, if you increase [MAXCHANNELS](https://github.com/NVIDIA/nccl/blob/master/src/include/devcomm.h#L49).

**Comment by hamid-ramezani** [comment]
@sjeaugey Thank you very much!

**Comment by hamid-ramezani** [comment]
Hi again, 

I had a follow-up question about this. I recently changed [MAXCHANNELS](https://github.com/NVIDIA/nccl/blob/master/src/include/devcomm.h#L49) to 128. Also, I export  `NCCL_MIN_NCHANNELS=128` . After running nccl-tests I got this error message: `'too many blocks in cooperative launch'` 

I was wondering if there is any workaround for this? i.e., run nccl-tests with 128 blocks (and channels).  Thanks!
